### ISSUE TITLE
JetBlog: SmartPostTiles widget - WCAG compatibility

### ISSUE THREAD

**Issue by aksamitnaya** [CrocoBlock/suggestions#265]
When tested aging ADA compatibility (web accessibility) it shows an error  that goes against 2.4.4 Link Purpose (In Context) (Level A) standard. Since the whole box is clickable it shows to the screen reader as an empty link that contains no text. 

for example, we are using the Smart Post Tiles element from your plugin to display 3 posts in the row. However each of the whole tiles are clickable which means that for  screen reader it shows as a empty link. If a link contains no text, the function or purpose of the link will not be presented to the user. This can introduce confusion for keyboard and screen reader users. So in plain English an anchor element has an href attribute, but contains no text (or only spaces) and no images with alternative text and this is against the 2.4.4 criterion in WCGA and this is a Level A criterion (there are AA and AAA criterions) so very basic. To fix it would be to remove the empty link or provide text within the link that describes the functionality and/or target of that link. Here is a tool for testing those compliances http://wave.webaim.org .

**Comment by kaskad88** [comment]
Done. Available in JetBlog 2.1.15
